__tests__/index.test.js

```diff
@@ -25,6 +26,19 @@ describe('#fetch', () => {
     mock.done();
   });
 
+  it('should catch and toss invalid headers present in a HAR', async () => {
+    const mock = nock('https://httpbin.org')
+      .post('/post')
+      .reply(function () {
+        expect(this.req.headers['x-api-key']).toStrictEqual(['asdf1234']);
+        expect(this.req.headers['x-api-key (invalid)']).toBeUndefined();
```

Comment:
TIL this is an invalid header name: `x-api-key (invalid)`

Reply:
Interesting that curl doesnt appear to block it, but this looks to be an error from httpbin?

```
$ curl -H "x-api-key (invalid): test" https://httpbin.org/headers
<html>
  <head>
    <title>Bad Request</title>
  </head>
  <body>
    <h1><p>Bad Request</p></h1>
    Invalid HTTP header name: &#x27;X-API-KEY (INVALID)&#x27;
  </body>
</html>
```

Reply:
Postman fails on it too before you can make a request.

![Screen Shot 2021-07-29 at 4 49 48 PM](https://user-images.githubusercontent.com/33762/127579524-6a0be094-1520-4bb5-a3d7-bb140ff9b7f2.png)

Reply:
Can also see this locally:

![Screen Shot 2021-07-29 at 4 52 58 PM](https://user-images.githubusercontent.com/33762/127579728-a169c4e0-2e39-40c2-9e7b-571f2cbc5d7e.png)

cURL must not be doing header name validation? 🤷